### ISSUE TITLE
safe way to let WebStorm know 'runner' and 'webServer' ports

### ISSUE THREAD

**Issue by segrey** [karma-runner/karma#614]
It's a feature request for WebStorm integration.

WebStorm needs to know ```webServerPort``` and ```runnerPort```.
Now this information is mined from the standard output of Karma server:
```
INFO [karma]: Karma v0.9.4 server started at http://localhost:9876/
...
INFO [karma]: To run via this server, use "karma run --runner-port 9101"
```

Such approach seems to be unsafe.
For example, WebStorm integration isn't working if ```logLevel: config.LOG_DISABLE``` is set.

It'd be nice to have the following API on JavaScript level:
```
onRunnerPortBound(runnerPort)
onWebServerPortBound(webServerPort)
```
There functions should be called only once with really bounded ports.
```config.runnerPort``` and ```config.port``` aren't what's needed. These ports are only initial port numbers. If a port cannot be bound, it's attempted to bind the port number increased by one and so on.

Thanks!

**Comment by Dignifiedquire** [comment]
Closing due to inactivity. Please feel free to reopen a new issue with updated info if this is still a problem. Thanks.
